src/test/java/uk/nhs/prm/deduction/e2e/queue/QueueMessageHelper.java

```diff
@@ -97,6 +121,18 @@ private SqsMessage findMessageContaining(String substring) {
         return null;
     }
 
+    private ArrayList<SqsMessage> findAllMessagesContaining(String substring) {
+        var allMessages = thinlyWrappedSqsClient.readThroughMessages(this.queueUri, 180);
+        ArrayList<SqsMessage> messages = new ArrayList<>();
+        for (var message : allMessages) {
+            System.out.println("just finding message, checking conversationId: " + substring);
+            if (message.contains(substring)) {
+                messages.add(message);
+            }
+        }
```

Comment:
Can we change this to use foreach?
```suggestion
        allMessages.forEach(message -> {
            System.out.println("just finding message, checking conversationId: " + substring);
            if (message.contains(substring)) {
                messages.add(message);
            }
        });
```

Reply:
Additionally, can we use String.format to keep it consistant please?